src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -3458,6 +3458,17 @@ public ILocalSymbol GetDeclaredSymbol(CatchDeclarationSyntax catchDeclaration, C
                         break;
                     }
 
+                case BoundKind.ConditionalAccess:
+                    {
+                        var conditionalAccess = (BoundConditionalAccess)boundNode;
+                        if (conditionalAccess.AccessExpression is BoundExpression expression && expression.ExpressionSymbol is Symbol symbol)
+                        {
+                            symbols = ImmutableArray.Create(symbol);
```

Comment:
>symbols = ImmutableArray.Create(symbol); [](start = 28, length = 40)

I don't think we would want to do this. The client should dig into the ```ConditionalAccessExpressionSyntax.WhenNotNull``` instead when appropriate.

Reply:
Done.